main.cc

```diff
@@ -727,6 +728,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
     sharded<locator::effective_replication_map_factory> erm_factory;
     sharded<service::migration_notifier> mm_notifier;
     sharded<service::endpoint_lifecycle_notifier> lifecycle_notifier;
+    std::optional<utils::disk_space_monitor<lowres_clock>> disk_space_monitor;
```

Comment:
Why optional?

Reply:
Please add _shard0 so it's clear it's only usable there.

Reply:
To contro its construction, similar to sharded services.

Reply:
> Please add _shard0 so it's clear it's only usable there.

Done in v9 ([733ca78](https://github.com/scylladb/scylladb/commit/733ca78ff5d46303b1250c9c595f7668083a239d))